tests/std/tests/Dev11_0000000_tuple_cat/test.cpp

```diff
@@ -209,6 +209,9 @@ int main() {
         assert(make_from_tuple<int>(tuple<>{}) == 0);
 #endif // _HAS_CXX17
     }
+
+    // LWG-3211 std::tuple<> should be trivially constructible
+    STATIC_ASSERT(is_trivially_constructible_v<tuple<>>);

```

Comment:
This is correct, but I recommend using the more specific type trait `is_trivially_default_constructible_v` (mostly for being able to search the tests).
```suggestion
    STATIC_ASSERT(is_trivially_default_constructible_v<tuple<>>);
```